packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -27,6 +27,14 @@ import { Endpoint } from '@theia/core/lib/browser/endpoint';
 import { VSXEnvironment } from '../common/vsx-environment';
 import { VSXExtensionsSearchModel } from './vsx-extensions-search-model';
 import { VSXExtensionNamespaceAccess, VSXUser } from '../common/vsx-registry-types';
+import { CommandRegistry, MenuPath } from '@theia/core/lib/common';
+import { ContextMenuRenderer } from '@theia/core/lib/browser';
+
+export const EXTENSIONS_CONTEXT_MENU: MenuPath = ['extensions_context_menu'];
+
+export namespace VSXExtensionsContextMenu {
+    export const COPY = [...EXTENSIONS_CONTEXT_MENU, '1_copy'];
```

Comment:
I'm fine with having `1_` at the moment, it will likely need to be updated as we add new items (until it reaches `4_`).